### ISSUE TITLE
ios plugin downloading forever

### ISSUE THREAD

**Issue by dcpenteado** [mapsplugin/cordova-plugin-googlemaps#906]
Hi!

When I try to install this plugin for iOS the download never ends.

Command: cordova plugin add cordova-plugin-googlemaps --variable API_KEY_FOR_ANDROID="real_key_android" --variable API_KEY_FOR_IOS="real_key_ios"

Response: 
Installing "plugin.google.maps" for ios
Fetching plugin "https://bitbucket.org/nightstomp/cordova-plugin-googlemaps-sdk.git" via git clone

but never ends...

Can someone help me?

Tks!

**Comment by wf9a5m75** [comment]
Did you install `git` command? If no, install it, and try again.

**Comment by Hirbod** [comment]
a problem that happens sometimes. we can't do anything against that

**Comment by JeffreyJamesMMM** [comment]
Was a solution ever found for this? My iOS builds take a really long time because this plugin's dependency (cordova-plugin-googlemaps-sdk) takes several minutes to complete.

**Comment by masashiGMS** [comment]
Download zip file from github page.
https://github.com/mapsplugin/cordova-plugin-googlemaps-sdk

Then try to install the sdk plugin from local path

**Comment by JeffreyJamesMMM** [comment]
So then there's no solution then? It's the only plugin that hangs for a long time and I don't want to update out of band with how plugins are installed for my build process. Thanks for replying anyways.

**Comment by wf9a5m75** [comment]
Or reinstall the latest git command. This works sometimes.

**Comment by r-h36** [comment]
@JeffreyJamesAttenti , I tried @masashiGMS 's method, didn't work. Then I tried @wf9a5m75 's method, installed the recent version of git. It worked. 
The system I'm using is Mac OS Sierra.
Used to be default git installed by Xcode packages (this one didn't work).
Then installed the git v2.15.1 from Git-SCM.com (note, the 2.15.0 version had a bit of issue opening in Mac OS)
After installing git 2.15.1, you probably need to run 
`export PATH="/usr/local/bin:/usr/bin/git:/usr/bin:/usr/local/sbin:$PATH"`
 to replace the xcode git in shell path.
Hope this helps
